app/src/main/java/org/dhis2/usescases/teiDashboard/DashboardRepositoryImpl.kt

```diff
@@ -413,10 +418,30 @@ class DashboardRepositoryImpl(
                 getTeiHeader(),
                 getTeiProfilePath(),
                 getOwnerOrgUnit(teiUid),
+                getQuickActions(programUid),
             )
         }
     }
 
+    private fun getQuickActions(programUid: String): List<String> {
```

Comment:
Can you set this list to use an enum or an object data class (would be better)? Also, remove commented code :)